website_and_docs/content/documentation/grid/getting_started.en.md

```diff
@@ -15,7 +15,9 @@ aliases: [
 1. Prerequisites
     * Java 11 or higher installed
     * Browser(s) installed
-    * Browser driver(s) [installed and on the `PATH`]({{< ref "../webdriver/getting_started/install_drivers.md#2-the-path-environment-variable" >}})
+    * Browser driver(s)
```

Comment:
These shouldn't be multiple bullets, and "getting started" should be minimal explanation. recommend:

```suggestion
    * Browser driver(s) [installed and on the `PATH`]({{< ref "../webdriver/getting_started/install_drivers.md#2-the-path-environment-variable" >}}) (if using Selenium 4.5 or lower)
```

Reply:
(oh that showed up as an auto commit option; you'd have to also delete the next lines if you accept this.)

Reply:
I'd prefer to leave it more explicit in the docs.